web/beacon-app/vite.config.ts

```diff
@@ -16,4 +16,6 @@ export default defineConfig({
   build: {
     outDir: 'build',
   },
+  envPrefix: ["VITE_", "REACT_APP_"],
```

Comment:
This was the fix here; by default Vite only cares about environment variables prefixed with `VITE_` by adding the `REACT_APP_` prefix here, our environment variables would be loaded. See:

https://vitejs.dev/config/shared-options.html#envprefix

Reply:
Aha, nice find!

Reply:
@bbengfort  good catch